<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[openBrowser]]>
    </line>
    <line>
      <![CDATA[login]]>
    </line>
    <line>
      <![CDATA[createCustomer]]>
    </line>
    <line>
      <![CDATA[logOut]]>
    </line>
    <line>
      <![CDATA[closeBrowser]]>
    </line>
  </reporter-output>
  <suite started-at="2023-03-19T10:10:44 IST" name="Default suite" finished-at="2023-03-19T10:10:56 IST" duration-ms="11978">
    <groups>
    </groups>
    <test started-at="2023-03-19T10:10:44 IST" name="Default test" finished-at="2023-03-19T10:10:56 IST" duration-ms="11978">
      <class name="com.actitime.testscript.CustomModule">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:com.actitime.testscript.CustomModule@649bec2e]" started-at="2023-03-19T10:10:44 IST" name="openBrowser" finished-at="2023-03-19T10:10:46 IST" duration-ms="2229" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[openBrowser]]>
            </line>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.actitime.testscript.CustomModule@649bec2e]" started-at="2023-03-19T10:10:46 IST" name="login" finished-at="2023-03-19T10:10:51 IST" duration-ms="5193" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[login]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="createCustomer()[pri:0, instance:com.actitime.testscript.CustomModule@649bec2e]" started-at="2023-03-19T10:10:51 IST" name="createCustomer" finished-at="2023-03-19T10:10:51 IST" duration-ms="8" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at com.actitime.testscript.CustomModule.createCustomer(CustomModule.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[createCustomer]]>
            </line>
          </reporter-output>
        </test-method> <!-- createCustomer -->
        <test-method is-config="true" signature="logOut()[pri:0, instance:com.actitime.testscript.CustomModule@649bec2e]" started-at="2023-03-19T10:10:51 IST" name="logOut" finished-at="2023-03-19T10:10:56 IST" duration-ms="4241" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logOut]]>
            </line>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.actitime.testscript.CustomModule@649bec2e]" started-at="2023-03-19T10:10:56 IST" name="closeBrowser" finished-at="2023-03-19T10:10:56 IST" duration-ms="107" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[closeBrowser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.actitime.testscript.CustomModule -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
